### PR TITLE
Add toggle functionality

### PR DESCRIPTION
**Pull request by ChrisBaidoo**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,10 @@
-
+function toggleToDo(element){
+  var newClass;
```
Comment (samdemaeyer) [inline main.js]: @ChrisBaidoo, let's use `const` instead of `var` in this project

File: main.js [hunk 2]
```diff
@@ -1 +1,10 @@
...
+  var completed = element.querySelector(".fa").classList.contains("fa-check-square");
```
Comment (samdemaeyer) [inline main.js]: @ChrisBaidoo, could you refactor so we don't have to duplicate `element.querySelector(".fa")` 4 times.

File: main.js [hunk 3]
```diff
@@ -1 +1,10 @@
...
+  if (completed) {
+    newClass = element.querySelector(".fa").className.replace("fa-check-square", "fa-square-o");
```
Comment (samdemaeyer) [inline main.js]: @ChrisBaidoo, Could you find a way to do this in a different way, to not use an `if` statement?

File: main.js (hunk 1 above)
Comment (ChrisBaidoo) [reply main.js]: @samdemaeyer using "const" makes it difficult to change variable later. I have used "let" instead.

Comment (samdemaeyer) [reply main.js]: Good catch @ChrisBaidoo! `const` is indeed not a valid useage in this case, only `var` or `let` can be used. So let’s go for `let`

File: main.js (hunk 2 above)
Comment (ChrisBaidoo) [reply main.js]: @samdemaeyer I have created a var for the `element.querySelector(".fa")`  called faClassName 

`var faClassName = element.querySelector(".fa")`

File: main.js (hunk 3 above)
Comment (ChrisBaidoo) [reply main.js]: @samdemaeyer Great news, the task has been completed. Another good news is that I understand the concept of DOM. Happy days!

**Pull request merged**